worlds/tloz/Rules.py

```diff
@@ -58,22 +59,25 @@ def set_rules(tloz_world: "TLoZWorld"):
             add_rule(world.get_location(location.name, player),
                      lambda state: state.has("Stepladder", player))
 
+    # Level 4 Access
+    for location in tloz_world.levels[4].locations:
+        add_rule(world.get_location(location.name, player),
```

Comment:
I was like "wait, why is this `world.get_location` but it has `player`...
`world = tloz_world.multiworld` :(

Reply:
your comment confuses me.
 but also, this block has existed for a while, i just moved it to line up with the dungeon order

Reply:
`add_rule(world.get_location(location.name, player)`
This isn't using a `world` and thus isn't using [`world.get_location`](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/AutoWorld.py#L531-L532). I thought it *was* using it, because it looks like it is, but TLoZ is just naming something `world` despite it being a `multiworld`

Reply:
The comment is more just lamenting the continued existence of `world: multiworld` and other similar things. I don't think you need to change it, to be clear, but thought it was funny / that I should point it out if you didn't know

Reply:
ah. that part was written before the whole multiworld rework thingy. 
frankly, i don't have the spoons to update it to work "correctly" atm